templates/index.html

```diff
@@ -83,18 +116,214 @@ <h1>NOTAM Sorter</h1>
         		additionalDestinationsDiv.appendChild(destinationRow);
 		}
 
+
 		// Create Destination Location input
 		var destinationLocationInput = document.createElement('div');
 		destinationLocationInput.classList.add('col-md-6', 'mb-3');
+        // Added an id to each child to get its value for checking purposes.
 		destinationLocationInput.innerHTML = '<label for="destinationLocation' + destinationCounter + '" class="form-label">Destination Location ' + destinationCounter + '</label>' +
-			'<input type="text" class="form-control" name="destinationLocation' + destinationCounter + '">';
+			'<input type="text" class="form-control" id= "destinationLocation' + destinationCounter + '" name="destinationLocation' + destinationCounter + '" >';
+
 
 		// Append the destination input to the last destination row
 		additionalDestinationsDiv.lastChild.appendChild(destinationLocationInput);
 
+        // Create an error message child for the new destination.
+        var destinationErrorMsg = document.createElement('span');
+        destinationErrorMsg.id = 'error-message' + (destinationCounter + 3);
+
+        // Get the last destination location input in the row
+        var lastDestinationLocationInput = additionalDestinationsDiv.lastChild.lastElementChild;
+
+        // Insert the error message span right after the last destination location input
+        lastDestinationLocationInput.insertAdjacentElement('beforeend', destinationErrorMsg);
+
 		// Increment the destination counter for the next input
 		destinationCounter++;
 	}
 </script>
 
+
+
+<script>
+    // Function to reset the body of the page and change it to loading.
+    function submitForm() {
+        // Store the form data.
+        const formData = new FormData(document.getElementById('dataForm'));
+
+        // Remvoe all the text and input boxes
+        document.getElementById('Title').remove();
+
+        document.getElementById('effectiveStartDate').remove();
+        document.querySelector('label[for="effectiveStartDate"]').remove();
```

Comment:
Removing each element by its id seems redundant, but when trying to remove their parent element, the fetch request for 'dataForm' fails. (at least i think that is what is happening)

Reply:
yes.